src/material/menu/menu.spec.ts

```diff
@@ -1326,6 +1326,29 @@ describe('MatMenu', () => {
       expect(panel.classList).not.toContain('mat-menu-above');
     }));
 
+    it('should update the position classes if the window is resized', fakeAsync(() => {
```

Comment:
@crisbeto This test does somehow not cover the new logic as it passes even without the changes in code.